src/lib/notifications.ts

```diff
@@ -0,0 +1,44 @@
+import { notificationStore } from '../stores'
+import { uuid } from '$lib/common/utils'
+
+export type Notification = {
+  id?: string
+  msg?: string
+  type?: string
+  timeout?: number
+}
+
+export const removeNotification: (id: string) => void = id => {
+  notificationStore.update(all =>
+    all.filter(notification => notification.id !== id)
+  )
+}
+
+export const addNotification: (
+  msg: string,
+  type?: string,
+  timeout?: number
+) => void = (msg, type = 'info', timeout = 5000) => {
+  // uuid for each notification
+  const id = uuid()
+
+  // adding new notifications to the bottom of the list so they stack from bottom to top
+  notificationStore.update(rest => [
```

Comment:
Notifications will be stacked with the most recently added ones appearing at the bottom of the list